content/news/2024-06-28-mapme-announcement/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Mapme joins the MapLibre Sponsorship Program"
+date: "2024-06-28"
+categories: ["announcements"]
+authors: [wipfli]
+draft: false
+---
+
+Mapme joins the MapLibre Sponsorship Program as a Silver Sponsor.
+
+<a href="https://mapme.com/">
+<img src="maplibre-mapme-logo.png" width="100%"/>
+</a>
+
+<br />
+<br />
+
+The Silver tier [donation](https://opencollective.com/maplibre/contributions/771444) of USD 10,000 will be used for general maintenance of the MapLibre codebases and project coordination.
+
+<a href="https://mapme.com/">Mapme</a>, the versatile tool for creating interactive maps with no coding required, is thrilled to join the MapLibre Sponsorship Program to support open-source libraries and tools for maps.
```

Comment:
We should write the entire post from our perspective, so consider changing this to "We are thrilled"...?

Reply:
Good idea